docs/sources/reference/templating.md

```diff
@@ -347,6 +351,18 @@ This variable is only available in the Singlestat panel and can be used in the p
 
 Currently only supported for Prometheus data sources. This variable represents the range for the current dashboard. It is calculated by `to - from`. It has a millisecond and a second representation called `$__range_ms` and `$__range_s`.
 
+### The $__dashboard Variable
+> Only available in Grafana v6.6+
+
+This variable is the uid of the current dashboard.
```

Comment:
uid - Is this user ID?

Reply:
UID is Unique ID, a string like "1QWfqrLiz" that identify a dashboard. You could specify it when you import a dashboard. 
[https://grafana.com/docs/grafana/latest/reference/dashboard/](url)